keras_cv/layers/feature_pyramid.py

```diff
@@ -66,6 +66,12 @@ class FeaturePyramid(tf.keras.layers.Layer):
             with feature inputs and merged result from upstream levels. Default to None,
             and a `keras.Conv2D` layer with kernel 3x3 will be created for each pyramid
             level.
+        num_extra_level: int, extra level of pyramid on top of existing `max_level`
+            features. This is used in case like RetinaNet, described in
+            (https://arxiv.org/pdf/1708.02002). Those extra levels will use features
+            from the top most level as inputs.
```

Comment:
Do we have cases where for non-consecutive levels, i.e., [2, 3, 5, 7]? It doesn't sounds useful, just want to make sure consecutive levels is our assumption for all our workflows.

Reply:
I didn't see any use case like that. Note that the size diff between each levels are 2x, which means if user config skip levels, the connections between those levels need to be customized as well.

Reply:
Ok sounds good -- maybe we should document that somewhere (also as a general principle maybe we find sometime to figure all the hidden implementation assumptions to put that in README).
For this PR, let's leave as-is.